### ISSUE TITLE
fs.smbfs.SMBFS init fails due to share list request

### ISSUE THREAD

**Issue by bjudkewitz** [althonos/fs.smbfs#37]
The host I work with refuses to list the shares, so fs.smbfs.SMBFS _init_ fails because it asks for said list (in https://github.com/althonos/fs.smbfs/blob/master/fs/smbfs/smbfs.py#L276)

Does fs.smbfs.SMBFS have to request the share list upon _init_? I know the name of the share I am interested in, so this request unnecessarily breaks functionality for me.

**Comment by althonos** [comment]
I don't think it does, I'm trying a patch that doesn't scan shares unless absolutely required. If that works, I'll push a new release. (sorry about the delay).
